editor/editor_export_godot3.cpp

```diff
@@ -421,6 +421,7 @@ static const char *prop_renames[][2] = {
 	{ "window/title", "window_title" }, // Dialogs
 	{ "z/relative", "z_as_relative" }, // Node2D
 	{ "z/z", "z_index" }, // Node2D
+	{ "is_pressed", "pressed" }, // BaseButton
```

Comment:
The remaps are ordered alphabetically, so it would be nice if you can move this one up :)